app/views/content_items/gone.html.erb

```diff
@@ -14,7 +14,7 @@
 
     <% if @content_item.alternative_path.present? %>
       <p class="alternative">
-        Visit: <%= alternative_path_link(request, @content_item.alternative_path)%>
+        <%= t("gone.visit") %>: <%= alternative_path_link(request, @content_item.alternative_path)%>
```

Comment:
Do we want to include the `:` in the string? Do all languages use the same `:` character to describe an explanation?

Reply:
Good idea, I'll move this to the locale file